app-near/src/server/api/routers/answers.ts

```diff
@@ -0,0 +1,15 @@
+import { TRPCError } from "@trpc/server";
+import { createTRPCRouter, protectedProcedure } from "../trpc";
+import { countAnswsers } from "../../su/answers/count";
+import { z } from "zod";
+
+export const answersRouter = createTRPCRouter({
+  count: protectedProcedure
+    .input(z.number())
+    .query(({ ctx, input: surveyId }) => {
+      if (surveyId !== ctx.session.user.surveyId) {
+        throw new TRPCError({ code: "FORBIDDEN" });
+      }
+      return countAnswsers(surveyId);
```

Comment:
:orange_circle: Typo
```suggestion
      return countAnswers(surveyId);
```